src/mina-signer/src/rosetta.ts

```diff
@@ -20,6 +20,14 @@ function signatureFromHex(signatureHex: string): Signature {
   };
 }
 
+function signatureToHex(signature: Signature): string {
+  // TODO possibly check isOdd like public key to hex does?
```

Comment:
not necessary here - "isOdd" is part of a compressed public key, but not part of a signature